bakery/src/tasks/bake-book-metadata.js

```diff
@@ -23,9 +24,14 @@ const task = () => {
           dedent`
           exec 2> >(tee baked-book-metadata/stderr >&2)
           collection_id="$(cat book/collection_id)"
-          mkdir "baked-book-metadata/$collection_id"
-          cp "assembled-book-metadata/$collection_id/collection.assembled-metadata.json" "baked-book-metadata/$collection_id/collection.baked-metadata.json"
-        `
+          book_dir="baked-book/$collection_id"
+          target_dir="baked-book-metadata/$collection_id"
+          mkdir "$target_dir"
+          cp "$book_dir/collection.baked.xhtml" "$target_dir/collection.baked.xhtml"
+          cp "assembled-book-metadata/$collection_id/collection.assembled-metadata.json" "$target_dir/collection.assembled-metadata.json"
+          cd "$target_dir"
+          python /code/scripts/bake-book-metadata.py collection.assembled-metadata.json collection.baked.xhtml collection.baked-metadata.json
```

Comment:
Would it be safer, perhaps, to explicitly pass the script `"$collection_id"` just like you do for `disassemble-book.py` so we know both use the same source value as the key to put / get metadata?

Reply:
ok, this makes sense, and in the script I can verify if the collection_id matches what is retrieved from the baked_html. 

thank you